trio/_socket.py

```diff
@@ -244,7 +294,7 @@ async def getprotobyname(name):
 ################################################################
 
 
-def from_stdlib_socket(sock):
+def from_stdlib_socket(sock: _stdlib_socket.socket) -> _SocketType:
```

Comment:
Ideally, public functions shouldn't return private types, since users can't then annotate their own stuff with that. `SocketType` is kinda too empty to be useful though...